.ci/build.sh

```diff
@@ -9,7 +9,7 @@ if [[ -n "$GCI_TOKEN" ]]; then
   python manage.py cleanse_gci_task_data private _site
   rm -rf private/
 else
-  python manage.py fetch_old_gci_task_data _site || true
+  python manage.py fetch_old_gci_task_data private || true
```

Comment:
oh, dont touch this :P

Reply:
Builds won't work without API token then?